libs/application/ui-fields/src/lib/FindVehicleFormField/FindVehicleFormField.tsx

```diff
@@ -305,6 +357,55 @@ export const FindVehicleFormField: FC<React.PropsWithChildren<Props>> = ({
                 focused={true}
               />
             )}
+            {energyDetails && !vehicleNotFound && (
+              <ActionCard
+                heading={`${energyDetails.make ?? ''} - ${
+                  energyDetails.permno
+                }`}
+                text={`${energyDetails.color} - ${
+                  energyFundsMessages &&
+                  formatText(
+                    energyFundsMessages.registrationDate,
+                    application,
+                    formatMessage,
+                  )
+                }: ${
+                  energyDetails.newRegistrationDate &&
+                  format(
+                    new Date(energyDetails.newRegistrationDate),
+                    'dd.MM.yyyy',
+                  )
+                }`}
+                tag={{
+                  label:
+                    (!energyDetails.hasReceivedSubsidy
+                      ? energyFundsMessages &&
+                        formatMessage(
+                          energyFundsMessages.checkboxCheckableTag as MessageDescriptor,
+                          {
+                            amount: energyDetails.vehicleGrant
+                              ? `${formatIsk(energyDetails.vehicleGrant)}`
+                              : energyFundsMessages &&
+                                formatMessage(
+                                  energyFundsMessages.carNotEligable as MessageDescriptor,
+                                ),
+                          },
+                        )
+                      : energyFundsMessages &&
+                        formatText(
+                          energyFundsMessages.checkboxNotCheckable,
+                          application,
+                          formatMessage,
+                        )) ?? '',
```

Comment:
Lot of logic to have in the markup, maybe move this to a util function in another file

Reply:
Moved to another file